linux_os/guide/system/permissions/files/permissions_local_var_log/ansible/shared.yml

```diff
@@ -0,0 +1,27 @@
+# platform = multi_platform_sle
+# reboot = false
+# complexity = low
+# strategy = configure
+# disruption = low
+
+- name: "Find log files"
+  find:
+    paths: '/var/log/'
+    recurse: yes
+    patterns: '*'
+  register: log_files
+
+- name: "Configure permission for /var/log/"
```

Comment:
Is this something SLE specific? It also does not agree with fix command in rule description. Can you please explain?

Reply:
Hi, yes this is kind of specific since relies on `chkstat` and `/etc/permissions` infrastructure to define expected permissions per directory and then `chkstat` utility checks and fixes that rule.